pkg/process/config/config_test.go

```diff
@@ -361,3 +362,52 @@ func TestIsAffirmative(t *testing.T) {
 	assert.Nil(t, err)
 	assert.False(t, value)
 }
+
+func TestGetCheckURL(t *testing.T) {
```

Comment:
we should check for eu endpoints as well

Reply:
Can you explain why? TBH I don't think that's necessary. The endpoint domain itself doesn't matter that much here. IMO what is important here is to test we're properly handling the path (when it's absent in the config, when it's present with `/api/v1/collector`, and when it has an arbitrary path)